src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Getting Started Page section items</description>
+    <fields>
+        <fullName>Description_Label__c</fullName>
+        <description>The custom label api name used in the description on a section item on the Getting Started page</description>
```

Comment:
@diego-caballero @fparodi-salesforce @force2b 

Is this an accurate description of what this field is used for? If so, please replace the Description and InlineHelpText with this value: "The api name for the Description label of a section item on the Getting Started page."